EMail/CustomEMailFunctions.lib.sh

```diff
@@ -236,7 +236,7 @@ _CreateEMailContent_CEM_()
     fi
 
     ## Header-1 ##
-    cat <<EOF > "$cemTempEMailContent"
+    cat <<EOF >> "$cemTempEMailContent"
```

Comment:
Using one angle bracket is correct & intentional. When you start to add the email header the file would not exist & if it did the current syntax would result in replacing whatever the old contents were.